crates/bifrost/src/providers/replicated_loglet/sequencer/appender.rs

```diff
@@ -198,6 +208,8 @@ impl<T: TransportConnect> SequencerAppender<T> {
             }
         };
 
+        tracing::trace!(gray_list=%gray_list, spread=%spread, "Sending store wave");
```

Comment:
I've seen a mixed use of "graylist" and "gray_list". It'd be nice if we make them consistent. I personally prefer `graylist` but whatever you choose, let's make all occurrences of it consistent please.